code/addons/a11y/package.json

```diff
@@ -82,10 +81,17 @@
     "react-dom": "^18.2.0",
     "react-resize-detector": "^7.1.2",
     "resize-observer-polyfill": "^1.5.1",
-    "typescript": "^5.7.3"
+    "typescript": "^5.7.3",
+    "vitest-axe": "^0.1.0"
   },
   "peerDependencies": {
-    "storybook": "workspace:^"
+    "storybook": "workspace:^",
+    "vitest": "^2.1.1 || ^3.0.0"
```

Comment:
@valentinpalkovic Do we actually need this? Is it possible to make this a dev dependency also since all we're using from `vitest-axe` is the matchers and those will get prebundled in?

Reply:
You are totally right. The matchers themselves don't depend on vitest. We can drop the peer-dependency requirement entirely! Good catch.